src/common/ExportSettingsModal/utils/createExportReport.js

```diff
@@ -0,0 +1,155 @@
+import moment from 'moment';
+
+import { getFullName } from '@folio/stripes/util';
+import {
+  DATE_FORMAT,
+  calculateFundAmount,
+} from '@folio/stripes-acq-components';
+
+const DATE_TIME_FORMAT = 'YYYY-MM-DDTHH:mm';
```

Comment:
weird format, what is the purpose? If I remember correctly, we need the same output as on UI